content/soda-gettingstarted/ak-sk-using-api.md

```diff
@@ -0,0 +1,154 @@
+---
+title: Access Key, Secret Key Generation using APIs.
+description: This document describes how to generate Access Key and Secret Key (AK/SK) using APIs.
+weight: 20
+---
+This document describes how to generate Access Key and Secret Key (AK/SK) using APIs. The AK/SK can be used to authenticate with SODA Identity Service for executing any other APIs in SODA, thereafter.
+
+##### AK: access key ID, which is a unique identifier used in conjunction with a secret access key to sign requests cryptographically.
+  
+##### SK: secret access key used in conjunction with an AK to sign requests cryptographically. It identifies a request sender and prevents the request from being modified.
+
+{{% notice info %}}
+**_To use the multicloud service, an Access Key, Secret Key (AK/SK) must be generated before anything else. To do this you can follow the steps below:_**  <br />
+Assuming that the user have access to cURL, the API execution is displayed using cURL commands.
+{{% /notice %}}
+
+
+#### 1. Create the Token for the "admin" User.
+API URI Format : 'http://<HOST:PORT>/identity/v3/auth/tokens'
+
+```bash
+curl --location --request POST 'http://192.168.xx.yy/identity/v3/auth/tokens' \
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "auth": {
+    "identity": {
+      "methods": [
+        "password"
+      ],
+      "password": {
+        "user": {
+          "name": "admin",
+          "domain": {
+            "name": "Default"
+          },
+          "password": "password"
+        }
+      }
+    }
+  }
+}'
+```
+
+Similarly, tokens can be generated for any of the users. To generate token for any specific user, replace the "name": "admin" with the <user-name> of the "user" and the user's corresponding password. 
+
+#### 2. Generate the Scoped Tokens 
+API URI Format : 'http://<HOST:PORT>/identity/v3/auth/tokens'
```

Comment:
same here, what is PORT?... what's the use of this

Reply:
not needed , removed.